packages/config/src/headers.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const { resolve } = require('path')
+
+// @todo: use `util.isDeepStrictEqual()` after dropping support for Node 8
+const fastDeepEqual = require('fast-deep-equal')
```

Comment:
We should be ok to use `util.isDeepStrictEqual()` now since we've fully deprecated Node 8 👍

Reply:
Right!
I will do it in a follow-up PR since we are using `fast-deep-equal` in several places in both `@netlify/build` and `@netlify/config`. :+1:

Reply:
Done in https://github.com/netlify/build/pull/3415